README.md

```diff
@@ -9,27 +9,32 @@
 
 <a href="https://info.blackducksoftware.com/OpenSourceRookies2015.html" ><img src="/media/images/Rookies_Award_Badge.png" width="150" alt="Hygieia℠ is now BlackDuck 2015 OpenSource Rookie of the year"></a>
 
-### What is Hygieia℠
+### What is Hygieia℠?
 
 ##### Pronunciation:      hi-gee-ya (origin Greek)
-Hygieia℠ is a single, configurable, easy to use dashboard to visualize near real-time status of the entire delivery pipeline.
+Hygieia℠ is a single, configurable, easy-to-use dashboard to visualize near real-time status of the entire delivery pipeline.
 
-Please view a video below to see Hygieia℠ in action
+Watch the video below to see Hygieia℠ in action:
 
  [![Hygieia℠ Is AWESOME](/media/images/video-shot.png)](https://www.youtube.com/watch?v=SoNTA78j0tc "Hygieia Is AWESOME")
 
-We use tools for everything today and each tool has its own unique dashboard to present information. What is lacking is a near real time consolidated view of the end-to-end delivery pipeline.  This view and a continuous feedback loop are critical to the success of
-any DevOps organization.
+We use tools for everything today, and each tool has its own unique dashboard to present information. Capital One DevOps Engineers developed Hygieia after researching existing tools and finding:
```

Comment:
I updated the file to add the lines that had been deleted. I thought "What is lacking..." was a little confusing since now it appears to no longer be lacking, but maybe that's just me!